Doc/howto/enum.rst

```diff
@@ -459,6 +459,30 @@ sense to allow sharing some common behavior between a group of enumerations.
 (See `OrderedEnum`_ for an example.)
 
 
+.. _dataclass_support:
+
+Dataclass support
+-----------------
+
+When inheriting from a :func:`dataclass` the :func:`repr` omits the inherited
+class' name.  For example::
```

Comment:
```suggestion
When inheriting from a :class:`~dataclasses.dataclass`,
the :meth:`~Enum.__repr__` omits the inherited class' name.  For example::
```

* Fix the cross-reference to dataclass
* Format it as a class/object rather than as a callable
* Reference the Enum specific `__repr__` rather than the general `repr` builtin
* Add comma for readibility